### PR TITLE
fix: lint issues for bootprocess

### PR DESCRIPTION
**Pull request by sharad-sharma**
Should be merged together with https://github.com/browserstack/enigma-access-modules/pull/70

### CODE REVIEW

**Review by vedharish — CHANGES_REQUESTED** [review]

File: bootprocess/tests/test_views_helper.py [hunk 1]
```diff
@@ -87,47 +97,53 @@ def __init__(self, is_owner=False, approver="", status="", user=""):
             self.group = Group()
 
         def filter(self, user="", status="", group="", is_owner=False):
+            """ mocked method filter """
+            del user, status, group
```
[the remark this answers is not in the document]
Comment (sharad-sharma) [reply bootprocess/tests/test_views_helper.py:101]: Tried this way ```def filter(self, _, __, ___, is_owner=False):``` but it's not working for named arguments.

File: pylama.ini [hunk 2]
```diff
@@ -14,6 +14,9 @@ ignore = W0718,W0719,E1101
 [pylama:Access/accessrequest_helper.py]
 ignore = W1202,R901,R0911,C0302,R0914
 
+[pylama:bootprocess/tests/test_views_helper.py]
+ignore = R0903
```
[the remark this answers is not in the document]
Comment (sharad-sharma) [reply pylama.ini:18]: Sure, adding it for `*/tests/test_*`.

**Comment by sharad-sharma** [comment]
lint is failing because of change in `emailSES` to `email_via_smtp` in `bootprocess.general`.
Should be passing for `fix_linter_issues_clone` branch of `enigma-access-modules`

File: Access/base_email_access/access.py [hunk 3]
```diff
@@ -205,3 +205,6 @@ def fetch_access_approve_email(self, request, data):
 
     def fetch_access_request_form_path(self):
         return "base_email_access/accessRequest.html"
+
+    def email_via_smtp(self, destination, subject, body):
```
Comment (vedharish) [inline Access/base_email_access/access.py:209]: Can we add a comment on why we have added it like this? `def email_via_smtp` calling helper method of same name - might be confusing in future

Also can we move it from bootprocess.general to `scripts/helpers` or `Access/helpers` - check within team what makes more sense - we will keep all global helper methods (like reading json from file etc) in the same file

**Review by vedharish — CHANGES_REQUESTED** [review]

**Review by vedharish — CHANGES_REQUESTED** [review]
Also celery pod is getting some issues on these branches

**Comment by sharad-sharma** [comment]
> Also celery pod is getting some issues on these branches

Fixed the issues by merging from main

**Review by vedharish — APPROVED** [review]

**Pull request merged**